garak/interactive.py

```diff
@@ -113,27 +113,20 @@ def do_probe(self, args):
         try:
             if self._cmd.generator:
                 generator_module_name = self._cmd.generator.split(".")[0]
-                generator_class_name = self._cmd.generator.split(".")[1]
+                generator_name = generator_name
```

Comment:
?

Reply:
Good catch
```suggestion
                generator_name = self._cmd.generator
```
This is should be the value passed for `load_plugin` now that DEFAULT_CLASS resolution is handle for us.